src/diffusers/pipelines/blip_diffusion/blip_image_processing.py

```diff
@@ -0,0 +1,456 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
```

Comment:
nit: "2023".

For other reviewers, it would be useful if we could add a comment directly in the code explaining why we're adding a new module specifically for image processing that seems to contain a lot of copied-over functions from `transformers`.